slack-api-client/src/main/java/com/slack/api/methods/RequestFormBuilder.java

```diff
@@ -2553,4 +2556,12 @@ private static void setIfNotNull(String name, Object value, MultipartBody.Builde
         }
     }
 
+    private static String getJsonWithGsonAnonymInnerClassHandling(List<LayoutBlock> blocks){
```

Comment:
I'd add a comment here that explains _why_ we need this method, just as a hint to a maintainer reading through this code in the future (I have a bad memory so I probably won't remember the details of this problem in 6 months!). Maybe just a quick link to the GSON issue and a single sentence explanation.

Reply:
Sounds good. Adding a link to the gson project issue would be easier.

Reply:
Agreed, I added a comment with the link to the original issue